src/pages/guides/using/marketo/marketo-observability-data-stream-setup.md

```diff
@@ -228,12 +228,12 @@ the processing results could potentially be reported across multiple events.
         "quota": 65432100,
         "queue": {
             "high": {
-                "lagSeconds": "2023-08-14T17:31:00Z",
+                "lagSeconds": 15,
                 "requestBacklog": 1,
                 "recordBacklog": 100
             },
             "normal": {
-                "lagSeconds": "2023-08-14T17:31:00Z",
+                "lagSeconds": 135,
```

Comment:
The customObject change above looks fine.  The lagSeconds change is fine though we should be aware that we don't know if it's correct or not since DIS at this time is not sending us status events so we can't confirm the exact format that's being sent.  We can only confirm with our own test scripts which may not be perfect.  Either way it looks fine for now.